tools/lsp/ui/api.slint

```diff
@@ -59,39 +59,75 @@ export struct DropMark {
 }
 
 export global Api {
-    in property <[ComponentListItem]> known-components;
-    out property <ComponentItem> visible-component;
-    in property <[Diagnostics]> diagnostics;
-    in property <[Selection]> selections;
-    in-out property <DropMark> drop-mark;
-    in property <[string]> known-styles;
-    in property <bool> experimental: false;
-    in property <bool> show-preview-ui: true;
-    in property <component-factory> preview-area;
-    in property <string> status-text;
-    in-out property <string> current-style;
+    // # Properties
+    // ## General preview state:
+    in property <bool> experimental: false; // experimental features are available
```

Comment:
This kind of comment is fine as long as you promise never to run the code formatter because else they'd end up in the next line that that'd be confusing.